src/__tests__/__snapshots__/documenter.test.ts.snap

```diff
@@ -3103,6 +3103,14 @@ The text will also be added to the \`title\` attribute of the button.",
       "optional": true,
       "type": "boolean",
     },
+    Object {
+      "description": "Provides a reason why the button is disabled (only when \`disabled\` is \`true\`).
+If provided, the button becomes focusable.
+Applicable for all button variants, except link.",
```

Comment:
Currently `disabledReason` works on the `link` variant as well. Do we want to prevent that or rely on the builders to follow our guidance?